pages/_app.js

```diff
@@ -11,6 +13,33 @@ export async function fetcher(...args) {
 }
 
 export default function App({ Component, pageProps }) {
+  const [favoriteIDs, setFavoriteIDs] = useLocalStorageState("favorites", {
+    defaultValue: [],
+  });
+
+  // Fetch plants from mongoDB
+  const { data: plants, error, isLoading } = useSWR("/api/plants", fetcher);
+
+  if (error) {
+    return <p>Could not fetch data!</p>;
```

Comment:
if i try to use npm run dev instead of the vercel preview, i get the Could not fetch data, i guess this makes sense cuz i dont have the env. local.
Still, the "Could not fetch data" is the only thing rendered on a blank html document. maybe you want to change this.

![image](https://github.com/SabinePfanner/plants-app/assets/123096081/88f47381-514b-463d-844c-f00aceb72bfd)

Reply:
Eva is trying to set up a test database so that you can hopefully test future iterations locally.